src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -0,0 +1,306 @@
+                    if (!String.IsNullOrEmpty(UpdateAsyncArgumentsParse(repositoryBranchUpdateHistory.Arguments)))
+                    {
+                        description.Append(UpdateAsyncArgumentsParse(repositoryBranchUpdateHistory.Arguments));
+                        break;
+                    }
+                    return;
+                // since this is not really an error, skipping this method
+                case "SynchronizePullRequestAsync":
+                    return;
+                // for all the other methods
+                default:
+                    description.Append("Repository :" + repositoryBranchUpdateHistory.Repository);
+                    break;
+            }
+            description.Append(Environment.NewLine);
+            description.Append("Branch Name : " + repositoryBranchUpdateHistory.Branch);
+            description.Append(Environment.NewLine);
+            description.Append("Error Message : " + repositoryBranchUpdateHistory.ErrorMessage);
+            description.Append(Environment.NewLine);
+            description.Append("Method : " + repositoryBranchUpdateHistory.Method);
+            description.Append(Environment.NewLine);
+            description.Append("Action : " + repositoryBranchUpdateHistory.Action);
+            description.Append(Environment.NewLine);
+            description.Append("/FYI : " + fyiHandles);
+            description.Append(Environment.NewLine + Environment.NewLine + Environment.NewLine);
+
+            // Get repo info used for create/ update gitHub issue
+            Octokit.Repository repo = await client.Repository.Get(_options.Owner, _options.Repository);
+            try
+            {
+                using (ITransaction tx = _stateManager.CreateTransaction())
+                {
+                    if (await gitHubIssueEvaluator.ContainsKeyAsync(tx, (repositoryBranchUpdateHistory.Repository, repositoryBranchUpdateHistory.Branch)))
+                    {
+                        // issue exists so just update the issue
+                        var issueNumber = await gitHubIssueEvaluator.TryGetValueAsync(tx, (repositoryBranchUpdateHistory.Repository, repositoryBranchUpdateHistory.Branch));
+                        Logger.LogInformation("Updating a gitHub issue number : " + issueNumber + " for the error : " + repositoryBranchUpdateHistory.ErrorMessage +
+                            " for the repository : " + repositoryBranchUpdateHistory.Repository);
+                        if (issueNumber.HasValue)
+                        {
+                            try
+                            {
+                                var issue = await client.Issue.Get(_options.Owner, _options.Repository, issueNumber.Value);
+                                var issueUpdate = issue.ToUpdate();
+                                issueUpdate.Body = issue.Body + Environment.NewLine 
```

Comment:
Why all the "Environment.NewLine". That's not really appropriate for web method. The newline of the running computer shouldn't affect the value we are sending to the server.  Just use the literal `\n`.

Reply:
I removed the newline with '\n'